virttest/utils_test/__init__.py

```diff
@@ -2151,6 +2151,7 @@ def unload_stress(self):
         """
         def _unload_stress():
             self.sendline(self.stop_cmd)
+            time.sleep(2)
```

Comment:
why do we need this ?

Reply:
As mentioned in PR message `In case of stopping stress app, in some cases have seen delays to stop stress app, giving couple of seconds sleep after stop_cmd fixes the issue`.
Even I have tried passing higher delay for first value in wait_for function below.. strangely it as well does not resolve the issue. The test use to wait for 2 secs after issuing stop command.. then find stress running still.. would attempt to sleep for couple of seconds and try executing func again.. but this time it would find stress not running... and test would fail with this reason.. when I tried sleep after stop_cmd I could see desired result all the time.. do you have any better solution here?

Reply:
I got you, but this is unnecessary because we do exactly the same thing with current flow,
in unload_stress() we use `first=2.0` as an argument to `wait_for()` that does this., 
https://github.com/avocado-framework/avocado-vt/blob/8372643276fcc6d7b2bd3fabc16f540377ce21d9/virttest/utils_test/__init__.py#L2161-L2162


with the below change I think it should work fine,

```patch
diff --git a/virttest/utils_test/__init__.py b/virttest/utils_test/__init__.py
index c8f00fa0..eb03a894 100755
--- a/virttest/utils_test/__init__.py
+++ b/virttest/utils_test/__init__.py
@@ -2153,10 +2153,11 @@ class Stress(object):
         stop stress tool manually
         """
         def _unload_stress():
-            self.sendline(self.stop_cmd)
-            if not self.app_running():
+            try:
+                self.sendline(self.stop_cmd)
+            except (process.CmdError, aexpect.ShellError):
                 return True
-            return False
+            return not self.app_running():
 
```

Reply:
i had mentioned about same in my previous reply... i tried same...  strangely it as well does not resolve the issue.

Reply:
I see proposed change introduce try catch, flags if cmderror. I am not sure if this proposed change would resolve issue as I had tried increasing `first` to 4 secs while calling wait_for . But would try same and get back

Reply:
@balamuruhans my bad. I overlooked and had interchanged first and step delays .. original step delay of 1sec seems not enough for stopping stress app on host. I am bumping it to 2 secs. This should remove the sleep call which I had introduced. Testing this change now.. once done will push same.